packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -957,6 +1002,11 @@ class SliverPhysicalParentData extends ParentData {
   /// top left visible corner of the sliver.
   Offset paintOffset = Offset.zero;
 
+  /// The [crossAxisFlex] factor to use for this sliver child.
+  ///
+  /// If null or zero, the child is inflexible and determines its own size in the cross axis.
+  int? crossAxisFlex;
```

Comment:
Have you tried this with the SliverGroup? I think I recall there was an issue with wanting to flex normal slivers along with SliverCross..Expanded. How will the group determine that a normal sliver, say SliverList, should flex if it's flex value is null?

Reply:
I think this is the only item left here to resolve. :)

Reply:
Yep I've tried this with SliverGroup and it works well. Essentially `SliverGroup`'s `setupParentData` method will create children `parentData` with `crossAxisFlex` of 1. However, the `ParentDataWidget`s for the `SliverCrossAxisConstrained` will set this to 0, for the constrained widgets.